psi4/driver/driver.py

```diff
@@ -1493,8 +1495,8 @@ def hessian(name, **kwargs):
 
         # Assemble Hessian from gradients
         #   Final disp is undisp, so wfn has mol, G, H general to freq calc
-        H = core.fd_freq_1(molecule, gradients, irrep)  # TODO or moleculeclone?
-        wfn.set_hessian(H)
+        H = driver_findif.comp_hess_from_grad(molecule, gradients, irrep)  # TODO or moleculeclone?
+        wfn.set_hessian(core.Matrix.from_array(H))
         wfn.set_gradient(G0)
         wfn.set_frequencies(core.get_frequencies())
```

Comment:
I think the global get_/set_frequencies in [`P::e`](https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libpsi4util/process.h#L60) can be killed off at this happy juncture. The wfn setting is already happening in `def freq`

Reply:
`wfn.set_frequencies` is called in `def freq`, yes. `wfn.set_hessian` and `wfn.set_gradient` are not. I'll strip the set_frequencies call in `def hessian`, but keep the others.

And yes, delighted to formally kill off some globals while I'm at it.